code-challenges.js

```diff
@@ -41,23 +81,83 @@ var vowelTester1 = "learn"
 var vowelTester2 = "throw"
 // Expected output: 3
 
+//function indexVowel logs the index of the first vowel of the string
+const indexVowel = (string) => {
+  //establish var vowels  [a, e, i, o, u]
+  var vowels = ["a", "e", "i", "o", "u"]
+  // split (string) into array
+let splitString = (string.split(""))
+
+// let indexOfFirstVowel1 =
+// 0;
+// for (let i = 0; i < splitString.length; i++) {
+//   if(vowels.includes(splitString[i].toLowerCase())) {
+//     indexOfFirstVowel = i;
+//     break;
+//   }
+
+// ALTERNATIVE NO FOR LOOP
+
+let arrayOfVowelsOnly = splitString.filter(letter => vowels.includes(letter))
+//map the array and return the first index of
+    // revised
+let indexOfFirstVowel = string.indexOf(arrayOfVowelsOnly[0])
+console.log(indexOfFirstVowel);
+}
+console.log(indexVowel(vowelTester1));
```

Comment:
good job! some minor indentation issues here and there but overall i can see very clearly what your approach waas and how you were refactoring along the way.